.github/workflows/e2e-ui-tests.yml

```diff
@@ -45,12 +45,29 @@ jobs:
        run: cp apps/backend/test/.env-ci apps/backend/.env
 
      - name: Create/migrate db
+       env:
+        NODE_ENV: test
        run: |
         yarn backend sequelize-cli db:create
         yarn backend sequelize-cli db:migrate
+        yarn backend sequelize-cli db:seed:all
 
-     - name: Build frontend and server for E2E UI tests
-       run: yarn run build
+     - name: Build Heimdall
+       env:
+        NODE_ENV: development
```

Comment:
Does the environment need to be set for every stage? Couldn't it just be set at the top? Also is there a reason that the build needs to happen in dev mode?

Reply:
The tests need to be seeded in test ENV to seed the policies into the database, and the site needs to be built in development mode as building it in test mode gives this: 
![image](https://user-images.githubusercontent.com/66680985/101816485-1478fc00-3aef-11eb-931d-e307112f0465.png)

Global env should work as test, with only dev as build, though.